modules/microshift-firewall-apply-settings.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-applying-settings_{context}"]
+== Applying firewall settings
+After you have finished configuring, run the following command to restart the firewall and apply settings:
```

Comment:
We already do this in the previous step, at the end, when we reload the firewall. Should it be done twice?

Reply:
Nope. I moved the previous into the configuring section. Better.